torchci/lib/bot/utils.ts

```diff
@@ -107,6 +107,35 @@ export class CachedIssueTracker extends CachedConfigTracker {
   }
 }
 
+export class CachedLabelerConfigTracker extends CachedConfigTracker {
+  repoLabels: any = {};
+  constructor(app: Probot) {
+    super(app);
+    app.on("push", async (context) => {
+      if (
+        context.payload.ref === "refs/heads/master" ||
+        context.payload.ref === "refs/heads/main"
+      ) {
+        await this.loadLabelsConfig(context, /* force */ true);
+      }
+    });
+  }
+
+  async loadLabelsConfig(context: Context, force = false): Promise<object> {
+    const key = repoKey(context);
+    if (!(key in this.repoLabels) || force) {
+      const config: any = await this.loadConfig(context, force);
+
+      if (config != null && "labeler_config" in config) {
```

Comment:
Just curious, I guess `labeler_config` on this context will come from https://github.com/pytorch/pytorch/blob/main/.github/labeler.yml.  Do you know how it happens?  Does GH does that for you when it sends the payload over to the bot?

Reply:
Ah, spot your PR description, it's magic

> Downloads the config based on a "labeler_config" key in .github/pytorch-probot.yml. By nature of context.config, it looks in .github/ + configfile for the config file